ERROR - 2025-03-21 13:58:16 --> mysqli_sql_exception: Table 'gritechs_fundacion_db.pagos' doesn't exist in C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `nro_com...', 0)
#1 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nro_com...')
#2 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nro_com...')
#3 C:\xampp2\htdocs\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `nro_com...', Array, false)
#4 C:\xampp2\htdocs\ci4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\ci4\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\ci4\app\Models\AportesModel.php(18): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\ci4\app\Controllers\AportesController.php(31): App\Models\AportesModel->getLastNumComp()
#8 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(933): App\Controllers\AportesController->mostrarFormulario()
#9 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AportesController))
#10 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp2\htdocs\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp2\htdocs\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp2\htdocs\ci4\system\rewrite.php(44): require_once('C:\\xampp2\\htdoc...')
#15 {main}
CRITICAL - 2025-03-21 13:58:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'gritechs_fundacion_db.pagos' doesn't exist
[Method: GET, Route: aportes/formulario]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `nro_comp`
FROM `pagos`
ORDER BY `nro_comp` DESC
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\AportesModel.php(18): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\AportesController.php(31): App\Models\AportesModel->getLastNumComp()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AportesController->mostrarFormulario()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AportesController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp2\\htdocs\\ci4\\public\\index.php')
CRITICAL - 2025-03-21 13:58:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'gritechs_fundacion_db.pagos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nro_comp`
FROM `pagos`
ORDER BY `nro_comp` DESC
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nro_comp`
FROM `pagos`
ORDER BY `nro_comp` DESC
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `nro_comp`
FROM `pagos`
ORDER BY `nro_comp` DESC
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\AportesModel.php(18): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\AportesController.php(31): App\Models\AportesModel->getLastNumComp()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AportesController->mostrarFormulario()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AportesController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp2\\htdocs\\ci4\\public\\index.php')
CRITICAL - 2025-03-21 13:58:17 --> [Caused by] mysqli_sql_exception: Table 'gritechs_fundacion_db.pagos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `nro_comp`
FROM `pagos`
ORDER BY `nro_comp` DESC
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nro_comp`
FROM `pagos`
ORDER BY `nro_comp` DESC
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nro_comp`
FROM `pagos`
ORDER BY `nro_comp` DESC
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `nro_comp`
FROM `pagos`
ORDER BY `nro_comp` DESC
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\AportesModel.php(18): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\AportesController.php(31): App\Models\AportesModel->getLastNumComp()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AportesController->mostrarFormulario()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AportesController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp2\\htdocs\\ci4\\public\\index.php')
ERROR - 2025-03-21 13:58:40 --> mysqli_sql_exception: Table 'gritechs_fundacion_db.usuarios' doesn't exist in C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\ci4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\ci4\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\ci4\app\Models\AuthModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\ci4\app\Controllers\Auth.php(30): App\Models\AuthModel->check_login('borisfavio@gmai...', '701120')
#8 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(933): App\Controllers\Auth->do_login()
#9 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp2\htdocs\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp2\htdocs\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp2\htdocs\ci4\system\rewrite.php(44): require_once('C:\\xampp2\\htdoc...')
#15 {main}
ERROR - 2025-03-21 13:58:40 --> Error en check_login: Table 'gritechs_fundacion_db.usuarios' doesn't exist
ERROR - 2025-03-21 13:58:52 --> mysqli_sql_exception: Table 'gritechs_fundacion_db.usuarios' doesn't exist in C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\ci4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\ci4\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\ci4\app\Models\AuthModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\ci4\app\Controllers\Auth.php(30): App\Models\AuthModel->check_login('aapaza@gmail.co...', 'aapaza')
#8 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(933): App\Controllers\Auth->do_login()
#9 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp2\htdocs\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp2\htdocs\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp2\htdocs\ci4\system\rewrite.php(44): require_once('C:\\xampp2\\htdoc...')
#15 {main}
ERROR - 2025-03-21 13:58:52 --> Error en check_login: Table 'gritechs_fundacion_db.usuarios' doesn't exist
ERROR - 2025-03-21 13:59:33 --> mysqli_sql_exception: Table 'gritechs_fundacion_db.usuarios' doesn't exist in C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\ci4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\ci4\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\ci4\app\Models\AuthModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\ci4\app\Controllers\Auth.php(30): App\Models\AuthModel->check_login('aapaza@gmail.co...', 'aapaza')
#8 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(933): App\Controllers\Auth->do_login()
#9 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp2\htdocs\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp2\htdocs\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp2\htdocs\ci4\system\rewrite.php(44): require_once('C:\\xampp2\\htdoc...')
#15 {main}
ERROR - 2025-03-21 13:59:33 --> Error en check_login: Table 'gritechs_fundacion_db.usuarios' doesn't exist
ERROR - 2025-03-21 14:02:17 --> mysqli_sql_exception: Table 'gritechs_fundacion_db.usuarios' doesn't exist in C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\ci4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\ci4\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\ci4\app\Models\AuthModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\ci4\app\Controllers\Auth.php(30): App\Models\AuthModel->check_login('borisfavio@gmai...', '701120')
#8 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(933): App\Controllers\Auth->do_login()
#9 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp2\htdocs\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp2\htdocs\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp2\htdocs\ci4\system\rewrite.php(44): require_once('C:\\xampp2\\htdoc...')
#15 {main}
ERROR - 2025-03-21 14:02:17 --> Error en check_login: Table 'gritechs_fundacion_db.usuarios' doesn't exist
ERROR - 2025-03-21 14:21:40 --> mysqli_sql_exception: Table 'gritechs_fundacion_db.usuaris' doesn't exist in C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp2\htdocs\ci4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\ci4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\ci4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\ci4\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\ci4\app\Models\AuthModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\ci4\app\Controllers\Auth.php(30): App\Models\AuthModel->check_login('borisfavio@gmai...', '701120')
#8 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(933): App\Controllers\Auth->do_login()
#9 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp2\htdocs\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp2\htdocs\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp2\htdocs\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp2\htdocs\ci4\system\rewrite.php(44): require_once('C:\\xampp2\\htdoc...')
#15 {main}
ERROR - 2025-03-21 14:21:40 --> Error en check_login: Table 'gritechs_fundacion_db.usuaris' doesn't exist
